form-app/src/components/formBody.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
+import '../style/formBody.css'
+import image from '../images/shecodes.jpg'
+import axios from "axios"
+import {nanoid} from "nanoid"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEnvelope, faMapPin, faUser } from '@fortawesome/free-solid-svg-icons'
+
+const FormBody = () => {
+    const [form, setForm] = useState({ name: '', email: '', subject: '', message: '' })
+    const handelChange = (event) => {
+        setForm({
+            ...form,
+            [event.target.id]: event.target.value,
+        });
+    };
+
+const handelSubmit = (event) => {
+    const post = {
+        id: nanoid(),
+        name: form.name,
+        email: form.email,
+        subject: form.subject,
+        message: form.message
+    };
+    axios.post(`https://my-json-server.typi.com/tundeojediran/contacts-api-server/inquiries`, post)
+        .then((res) => {
+            console.log(res.data)
+        }).catch((error) => {
+            console.log(error)
+        });
+    setForm({
+        name: '',
+        email: '',
+        message: ''
+    })
+    alert('Submission Successful!')
```

Comment:
Lines 32-37 should happen inside the `then` method in line 27. So move them to start from line 28